src/ap/java/org/spongepowered/tools/obfuscation/MixinObfuscationProcessor.java

```diff
@@ -80,13 +80,7 @@ protected void postProcess(RoundEnvironment roundEnv) {
 
     @Override
     public SourceVersion getSupportedSourceVersion() {
-        try {
-            return SourceVersion.valueOf("RELEASE_8");
-        } catch (IllegalArgumentException ex) {
-            // Java 8 not supported
-        }
-        
-        return super.getSupportedSourceVersion();
```

Comment:
This change here fixes gradle's annotation processor time tracking wrapper not complain about being unsupported. I dont see any reason for it to not always be the latest here? Please let me know if you have a better idea.